src/components/common/quantity/Quantity.tsx

```diff
@@ -1,9 +1,14 @@
-import React, { useCallback, useEffect, useState } from "react";
+import React, { useState } from "react";
 import { InputLabel } from "@madie/madie-design-system/dist/react/";
 import { Autocomplete, FormControl, TextField } from "@mui/material";
 import * as ucum from "@lhncbc/ucum-lhc";
 import KeyboardArrowDownIcon from "@mui/icons-material/KeyboardArrowDown";
 
+export interface UCUM {
+  label: string;
+  value: ucum;
+}
+
 export const textFieldStyle = {
```

Comment:
Why are we using TextField from Material UI instead of already styled component from Madie Design Systems  ?